src/index.html

```diff
@@ -14,56 +14,39 @@
   </style>
   <link rel="icon" href="./images/favicon/favicon.ico" sizes="32x32">
   <script async src="https://www.googletagmanager.com/gtag/js?id="></script>
+
   <script>
     window.dataLayer = window.dataLayer || [];
     function gtag() { dataLayer.push(arguments); }
     gtag('js', new Date());
 
     gtag('config', '');
   </script>
-  <!-- Facebook Pixel Code -->
-
-<script>
-
-!function(f,b,e,v,n,t,s)
-
-{if(f.fbq)return;n=f.fbq=function(){n.callMethod?
-
-n.callMethod.apply(n,arguments):n.queue.push(arguments)};
-
-if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
-
-n.queue=[];t=b.createElement(e);t.async=!0;
-
-t.src=v;s=b.getElementsByTagName(e)[0];
-
-s.parentNode.insertBefore(t,s)}(window,document,'script',
-
-'https://connect.facebook.net/en_US/fbevents.js');
-
 
-fbq('init', '2540911245998201'); 
+  <!--Linkedin Pixel-->
+  <img height="1" width="1" style="display:none;" alt="" src="https://px.ads.linkedin.com/collect/?pid=1643873&conversionId=1608193&fmt=gif" />
 
-fbq('track', 'PageView');
-
-</script>
-
-<noscript>
-
-<img height="1" width="1" 
-
-src="https://www.facebook.com/tr?id=2540911245998201&ev=PageView
-
-&noscript=1"/>
-
-</noscript>
-
-<!-- End Facebook Pixel Code -->
+  <!--Facebook Pixel-->
+  <script>
+    !function(f,b,e,v,n,t,s)
+    {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
+    n.callMethod.apply(n,arguments):n.queue.push(arguments)};
+    if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
+    n.queue=[];t=b.createElement(e);t.async=!0;
+    t.src=v;s=b.getElementsByTagName(e)[0];
+    s.parentNode.insertBefore(t,s)}(window,document,'script',
+    'https://connect.facebook.net/en_US/fbevents.js');
+    fbq('init', '551945405382177');
+    fbq('track', 'PageView');
+  </script>
+  <noscript>
+  <img height="1" width="1"
+    src="https://www.facebook.com/tr?id=551945405382177&ev=PageView&noscript=1"/>
```

Comment:
Why did this URL change? (it used to say `id=2540911245998201`)

Reply:
@QuantumArjun